### ISSUE TITLE
This project will not download audio from spotify

### ISSUE THREAD

**Issue by tobiaskellernetcom** [ToberoCat/SpotifyDownloader#2]
this projects wont download the tracks from spotify?!
it will get the tracks name from spotify and searches it on youtube, and starts than a download from youtube

**Comment by ToberoCat** [comment]
Yeah, because Spotify doesn't want you to Dow load the songs without premium - That's part of their business model, so going to a platform allowing you to download the video / audio is the only option for people without a subscription.

Therefore, this tool just speeds up this process
